src/screens/ItemEditScreen.tsx

```diff
@@ -39,7 +39,8 @@ export default function ItemEditScreen(props: PropsType) {
   const [loading, setLoading] = React.useState(true);
   const [content, setContent_] = React.useState("");
   const viewSettings = useSelector((state: StoreState) => state.settings.viewSettings);
-  const { viewMode } = viewSettings;
+  const { defaultViewMode = "last", lastViewMode = false } = viewSettings;
+  const [viewMode, setViewMode] = React.useState(defaultViewMode === "last" ? lastViewMode : defaultViewMode === "viewer" ? true : false);
```

Comment:
First comment: no need to use inline if for true/false, so you can drop that.
Second comment: could you please add parenthesis here around the main else branch and around conditions? I know it may not be the most common convention in the JS world, but I think this is much more readable:
```
(defaultViewMode === "last") ? lastViewMode : (defaultViewMode === "viewer")
```